src/control/IntersectionReferenceSystem.ts

```diff
@@ -67,7 +69,7 @@ export class IntersectionReferenceSystem {
 
   private setPath(path: number[][], options: ReferenceSystemOptions = {}): void {
     this.options = options;
-    const { trajectoryAngle } = this.options;
+    const { trajectoryAngle, trajectoryAngleStart } = this.options;
```

Comment:
trajectoryAngleEnd would probably be better.

Reply:
I removed the trajectoryAngleStart as it din't make much sense.